README.md

```diff
@@ -57,12 +57,12 @@ section in `recipes.mdwn` file.
 1. Create a new page with your tutorial/setup/widget/snippet in Markdown, with a
    relevant name, under the `recipes` folder.
 
-    Example `recipes/xrandr-tutorial.mdmw`
+    Example `recipes/xrandr-tutorial.mdwn`
 
 1. Link your page to the right section in the `recipes.mdwn` page with Markdown
    syntax.
 
-    Example `[XrandR Tutorial](../recipes/xrandr-tutorial.html)`
+    Example `[XrandR Tutorial](../recipes/xrandr-tutorial)`
```

Comment:
Urgh. That's where this comes from.

Could you please change this to (and check whether this actually works):
```
Example `[[XRandR Tutorial|recipes/xrandr-tutorial]]`
```

Thanks a lot for spotting and fixing this!